doc/guides/doc-style-guide.md

```diff
@@ -30,7 +30,29 @@
 * When documenting APIs, update the YAML comment associated with the API as
   appropriate. This is especially true when introducing or deprecating an API.
 * For code blocks:
-  * Use [language aware fences][]. (<code>```js</code>)
+  * Use [language][]-aware fences. (<code>```js</code>)
+  * For the [info string][], use one of the following.
+
+    | Meaning       | Info string       |
+    | ------------- | ----------------- |
+    | Bash          | `bash`            |
+    | C             | `c`               |
+    | C++           | `cpp`             |
+    | CoffeeScript  | `coffee`          |
+    | Diff          | `diff`            |
+    | HTTP          | `http`            |
+    | JavaScript    | `js`              |
+    | JSON          | `json`            |
+    | Markdown      | `markdown`        |
+    | Plaintext     | `text`            |
+    | Powershell    | `powershell`      |
+    | R             | `r`               |
+    | Shell Session | `console`         |
+
+    Note: If one of your language-aware fences needs an info string that is not
```

Comment:
```suggestion
    If one of your language-aware fences needs an info string that is not
```

Reply:
Good catch — fixup in https://github.com/nodejs/node/pull/34024/commits/14fd89b2100b5c374fda649787d2bb7f1c22833b.